guides/RELEASE-PROCESS.md

```diff
@@ -1,55 +1,74 @@
 # Release process
 
 1. **Create a release commit**
-   1. Run prepublish script from the root of your local git repo. This will bump the version, build the documentation, and transpile + bundle `dist` assets.
+
+    1. Get the latest from the `master` branch
+         ```
+        git pull
+         ```
+    1. Check the status of your working directory to see which changes have been staged, which haven’t, and which files aren’t being tracked by Git.
+         ```
+        git status
+         ```
+   1. Run the test suite and ensure JS unit tests, JS linting, and Sass linting pass locally.
+         ```
+         yarn test
+         ```
+   1. Run the prepublish script from the root of your local git repo. This will bump the version, build the documentation, and transpile + bundle `dist` assets.
       ```
       ./scripts/prepublish.sh
       ```
 
-   1. Commit and push to `master`:
+   1. Add, commit and push to `master` with the new release number:
+      ```
+      git add --all
+      ```
+      ```
+      git commit -m "Release v1.1.0"
+      ```
```

Comment:
Since you're separating these blocks anyway, maybe you should use the space above or below to clarify that 1.1.0 is an example and needs to be changed.